R/build_quantile_ensemble.R

```diff
@@ -20,30 +23,57 @@
 
 build_quantile_ensemble <- function(
   forecast_data,
+  weights_df = NULL,
   method = c("median", "mean"),
   model_name,
   forecast_date,
   location_data = NULL,
   location_col_name = "fips"
 ) {
+  library(matrixStats)
```

Comment:
I think we should follow Hadley's recommendations for how to call the two functions that we need in matrixStats. 
https://r-pkgs.org/dependencies.html#import-r

This involves, if I understand correctly, specifying the two functions in the DESCRIPTION file, and then running Roxygen on the package so that the NAMESPACE file includes those functions explicitly.

Reply:
And then I think any explicit call to library or require is discouraged.